ports/stm32/boards/MIKROE_QUAIL/mpconfigboard.mk

```diff
@@ -0,0 +1,13 @@
+MCU_SERIES = f4
+CMSIS_MCU = STM32F427xx
+AF_FILE = boards/stm32f427_af.csv
+ifeq ($(USE_MBOOT),1)
```

Comment:
Did you test with mboot?

Reply:
No, I didn't. I've just tried to compile it and it fails. I'll disable it for now. I'll come back to it because I need it in my project.